restaurant-front/src/modules/admin/admin-service/admin.service.ts

```diff
@@ -0,0 +1,121 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { Area } from 'src/modules/shared/models/area';
+import { Employee } from 'src/modules/shared/models/employee';
+import { RestaurantTable } from 'src/modules/shared/models/restaurant-table';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdminService {
+  private headers = new HttpHeaders({ 'Content-Type': 'text' });
+  private headers2 = new HttpHeaders({
```

Comment:
Instead of headers2 rename it ot headersJson.

Reply:
renamed